scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -2666,11 +2765,22 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
       case tok: Ellipsis if tok.rank == 2 =>
         List(ellipsis(2, astInfo[Term.Param]))
       case _ =>
+        var parsedUsing = false
```

Comment:
having it here is better because this scope is only inside defined method.
For `parsedImplicits` it is a little more complex as you declare it globally but set inside function, very hard to read